images/fluentd/tests/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "test" {
+  digest = var.digest
+  script = "${path.module}/01-test.sh"
```

Comment:
Let's use `working_dir` so the test doesn't have to know about the structure of the repo above this directory.

Reply:
The files it is accessing are above this directory (`examples` is generally a sibling).  I think my preference would be to move `examples` under the test directory, but this confuses `monopod` right now.  I was hoping we could do some broader cleanup once we're off of `monopod matrix` 🤞 

I can change it, it just felt a little gross to specify a `working_dir` and then use `../examples` 🤢

Reply:
tl;dr I'd love to work towards relatively self-contained modules 🤞

Reply:
https://github.com/chainguard-images/images/issues/925